index.html

```diff
@@ -1235,53 +1235,35 @@
   uses</a> for ongoing discussion and iteration on this principle.
 </aside>
 
-In maintaining duties of [=duty of protection|protection=], [=duty of discretion|discretion=] and [=duty of loyalty|loyalty=], user
-agents should be conservative when sharing data that isn't needed to satisfy a user's immediate goals.
-
-On the Web, user agents and sites often communicate data outside what is needed to load and display a
-page. These <dfn data-lt="ancillary use">ancillary uses</dfn> include:
-
-* browser telemetry, including:
-  * performance measurements,
-  * measuring feature usage,
-  * debugging site, browser and networking issues, and
-  * detecting security problems or attacks;
-* site telemetry, including:
-  * performance measurements,
-  * analytics about usage and visitors,
-  * debugging site issues, and
-  * detecting security problems or attacks;
-* advertising and social media sharing, including:
-  * measurement,
-  * targeting,
-  * operating ad auctions,
-  * personalization, and
-  * fraud prevention;
-* performance, including caching and prefetching content; or
-* software updates.
-
-<dfn>Telemetry and analytics</dfn>, including browser telemetry and site telemetry, are a common
-cluster of ancillary uses.
-
-In some cases, mechanisms for sharing data are provided in order to minimize some other data
-collection mechanism identified as more intrusive or costly.
-
-Different users will want to share different kinds and amounts of data with websites. For some uses,
-some people would reasonably anticipate sharing such data or identify that use as a
-commonly-accepted contextual norm and some people would be willing to participate (for example,
-might have explicitly decided to contribute if they had the time and understanding to consider
-details about such a use) and some people would identify those uses as supporting their own goals.
-
-Aggregated or [=de-identified=] data is often more likely to be
-acceptable in these senses and often still contributes to the collective benefit while minimizing
-privacy threats to a particular individual (see <a href="#principle-collective-privacy">collective
-privacy</a>). But some people might object anyway, in particular instances or in general.
+<div class="practice">
+<span class="practicelab">In maintaining duties of [=duty of protection|protection=], [=duty of discretion|discretion=] and [=duty of loyalty|loyalty=], user
+agents should take care that, when sharing data that isn't needed to satisfy a user's immediate goals, such information sharing
+aligns with the user's intents and interests.</span>
+</div>
+
+On the Web, user agents and sites can communicate data outside what is needed
+to load and display a page, data not needed to accomplish the user's intent on a site.
+This category of data is called <dfn data-lt="ancillary data">ancillary data</dfn>,
+and these uses are <dfn data-lt="ancillary use">ancillary uses</dfn> of data.
```

Comment:
```suggestion
Websites sometimes use data in ways that aren't needed for the user's immediate
goals. These uses are known as <dfn data-lt="ancillary use">ancillary uses</dfn>,
and data that is only useful for [=ancillary uses=] is <dfn>ancillary data</dfn>
```

Reply:
Or maybe "data that is primarily useful for ancillary uses":

```suggestion
Websites sometimes use data in ways that aren't needed for the user's immediate
goals. These uses are known as <dfn data-lt="ancillary use">ancillary uses</dfn>,
and data that is primarily useful for [=ancillary uses=] is <dfn>ancillary data</dfn>.
```

Reply:
i dig it